torch_frame/data/stats.py

```diff
@@ -39,7 +39,7 @@ class StatType(Enum):
 
     # text_embedded (Also, embedding)
     # Note: For text_embedded, this stats is computed in
-    # dataset.update_col_stats, not here.
+    # dataset._update_col_stats, not here.
```

Comment:
We can follow-up in the next PR, but we need to compute `EMB_DIM` as `compute_stats` here for `stype.embedding`.